addresslabel.html

```diff
@@ -67,7 +67,7 @@
 	</style>
 </head>
 <body>
-	{Logo}
+	{Logo(height: '100px')}
```

Comment:
Ei taida `Logo`-tagi tukea yksiköitä `width` ja `height`-attribuuteilla, eli `height: 100` voisi olla enemmän oikein. Saatan myös muistaa väärin. 😄

Reply:
Korjattu